build.include

```diff
@@ -14,7 +14,7 @@ IMAGE_TAG="next"
 THEIA_GITHUB_REPO="eclipse-theia/theia"
 THEIA_VERSION="master"
 THEIA_BRANCH="master"
-THEIA_COMMIT_SHA="2eb4319fb459f30ad897e8467878154c6790aaef"
+THEIA_COMMIT_SHA="d734315e505ad4a7691a3f6f4a07d2209af0d9cf"
```

Comment:
further adaptation will be done in https://github.com/eclipse-che/che-theia/pull/1124